_data/team/roles/community_engagement.yml

```diff
@@ -4,4 +4,7 @@ tasks:
   - Managing and triaging conversations on Discord and GitHub Discussions
 
 current_members:
-  - Jenna Swarthout Goddard
+  - "[Jenna Swarthout Goddard](https://github.com/jennaswa)"
+  - "[Yuxuan Zhuang](https://github.com/yuxuanzhuang)"
+  - "[Rocco Meli](https://github.com/RMeli)"
```

Comment:
@RMeli I added you here because you've done more than anyone else lately on GH Discussions. Are you ok to be listed here?

Reply:
Yes, that's fine, thanks.